README.md

```diff
@@ -8,8 +8,8 @@ For value transfers we recommend using [wallet.rs](https://github.com/iotaledger
 
 This library allows you to do the following:
 
-* Create messages with indexation and transaction payloads
-* Get messages and outputs
+* Create blocks with indexation and transaction payloads
```

Comment:
I know this isn't really part of your PR, but maybe change the sentence to:

"Create blocks with tagged data and transaction payloads"